adagrams/game.py

```diff
@@ -1,13 +1,101 @@
 from random import randint
-
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
 def draw_letters():
-    pass
+    pool = LETTER_POOL.copy()
+    list_letter = []
+    List_10 = []
+    for letter, count in pool.items():
+      list_letter += [letter] * count
+
+    print(list_letter)
+
+    for i in range(10):
+        random_index2 = randint(1, len(list_letter)) - 1
+        print(f"take index {random_index2}. current length {len(list_letter)}")
+        List_10.append(list_letter[random_index2])
+        list_letter.pop(random_index2) 
+    return List_10
 
 def uses_available_letters(word, letter_bank):
```

Comment:
The [PEP 8 style guide for blank lines](https://peps.python.org/pep-0008/#blank-lines) recommends 2 blank lines surround top level functions like these to give visual separation between functions that are not enclosed within a class.

```suggestion
    return List_10


def uses_available_letters(word, letter_bank):
```